edu-services/cm-service/cm-impl/hibernate-impl/hibernate/src/java/org/sakaiproject/coursemanagement/impl/CourseOfferingCmImpl.java

```diff
@@ -138,4 +138,20 @@ public String getStatus() {
 	public void setStatus(String status) {
 		this.status = status;
 	}
+
+	@Override
+	public String getTitle() {
+		if (title == null || title.equals(""))
+			return canonicalCourse.getTitle();
```

Comment:
Sakai adheres to Sun standards, and as such you need opening and closing braces on one line if/else statements. Leaving them open like this may be visually appealing, but can lead to problems down the road.